xml/x509_subjects.xml

```diff
@@ -0,0 +1,1019 @@
+        <example host.mac="4C4E315901D0" hw.product="RV180">CN=4C4E315901D0,OU=RV180,O=Cisco Systems\, Inc.,C=US</example>
+        <param pos="0" name="hw.device" value="Broadband Router"/>
+        <param pos="0" name="hw.vendor" value="Cisco"/>
+        <param pos="2" name="hw.product"/>
+        <param pos="1" name="host.mac"/>
+    </fingerprint>
+
+
+	<fingerprint pattern="^SERIALNUMBER=PID:([^ ]+) SN:([^,]+),CN=(?:[a-zA-Z0-9\-]+)-SEP([a-fA-F0-9]{12}),OU=CTG,O=Cisco Systems Inc\.$">
+        <description>Cisco / Linksys Router</description>
+        <example host.mac="B07D47D33A1C" hw.product="CP-8851" cisco.serial_number="FCH1924AHCA">SERIALNUMBER=PID:CP-8851 SN:FCH1924AHCA,CN=CP-8851-SEPB07D47D33A1C,OU=CTG,O=Cisco Systems Inc.</example>
+        <param pos="0" name="hw.device" value="IP Phone"/>
+        <param pos="0" name="hw.vendor" value="Cisco"/>
+        <param pos="1" name="hw.product"/>
+        <param pos="2" name="cisco.serial_number"/>
+        <param pos="3" name="host.mac"/>
+    </fingerprint>
+
+
+	<fingerprint pattern="^CN=Cambium WLAN AP,OU=Products,O=Cambium Networks Inc,L=San Jose,ST=CA,C=US$">
+        <description>Cambium Networks WAP</description>
+        <example>CN=Cambium WLAN AP,OU=Products,O=Cambium Networks Inc,L=San Jose,ST=CA,C=US</example>
+        <param pos="0" name="hw.device" value="WAP"/>
+        <param pos="0" name="hw.vendor" value="Cambium Networks"/>
+        <param pos="0" name="hw.product" value="WLAN AP"/>
+    </fingerprint>
+
+
+	<fingerprint pattern="^CN=([^,]+),OU=Products,O=Cambium Networks Inc,L=San Jose,ST=CA,C=US$">
+        <description>Cambium Networks Router</description>
+        <example hw.product="R190V">CN=R190V,OU=Products,O=Cambium Networks Inc,L=San Jose,ST=CA,C=US</example>
+        <example hw.product="R200">CN=R200,OU=Products,O=Cambium Networks Inc,L=San Jose,ST=CA,C=US</example>
+        <param pos="0" name="hw.device" value="Router"/>
+        <param pos="0" name="hw.vendor" value="Cambium Networks"/>
+        <param pos="1" name="hw.product"/>
+    </fingerprint>
+
+
+	<fingerprint pattern="^CN=Nepenthes Development Team,OU=anv,O=dionaea\.carnivore\.it,C=DE$">
+        <description>Nepenthes honeypot</description>
+        <example>CN=Nepenthes Development Team,OU=anv,O=dionaea.carnivore.it,C=DE</example>
+        <param pos="0" name="service.family" value="Nepenthes"/>
+        <param pos="0" name="service.product" value="Nepenthes"/>
+    </fingerprint>
+
+
+	<fingerprint pattern="^CN=IPMI,OU=Software,O=Super Micro Computer,ST=California,C=US$">
+        <description>Super Micro IPMI Controller</description>
+        <example>CN=IPMI,OU=Software,O=Super Micro Computer,ST=California,C=US</example>
+        <param pos="0" name="hw.device" value="Lights Out Management"/>
+        <param pos="0" name="hw.vendor" value="Super Micro"/>
+        <param pos="0" name="hw.product" value="IPMI"/>
+    </fingerprint>
+
+
+    <!-- iDRAC with a MAC -->
```

Comment:
for these (and perhaps other fingerprint groups) it would be better to put this differentiation in the `description` field as it will help more readily identify the difference between two fingerprints that would otherwise just be described as "Dell iDRAC Remote Access Controller"